e2e/bazel_managed_deps/WORKSPACE

```diff
@@ -25,6 +25,10 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.4.0/rules_nodejs-4.4.0.tar.gz"],
 )
 
+load("@build_bazel_rules_nodejs//nodejs:repositories.bzl", "rules_nodejs_dependencies")
```

Comment:
I've seen this pattern around. rules_docker uses it
```
load("@build_bazel_rules_nodejs//nodejs:repositories.bzl", rules_nodejs_dependencies = "dependencies")
rules_nodejs_dependencies()
```
It avoids the unnecessary scoping in the rnj repo. Wdyt?

Reply:
rules_docker:
```
load(
    "@io_bazel_rules_docker//repositories:repositories.bzl",
    container_repositories = "repositories",
)
container_repositories()
```

Reply:
I think it makes users use the re-assignment syntax in the load statement that they may not have seen, and is overall more complex to have a second name for the thing (some users might not re-assign)

Reply:
rules_docker kinda needs it since they have so many language ecosystems to namespace?